apps/labs/posts/pydata-extensibility-vision.md

```diff
@@ -0,0 +1,404 @@
+---
+title: 'A vision for extensibility to GPU & distributed support for SciPy, scikit-learn, scikit-image and beyond'
+published: November 17, 2021
+author: ralf-gommers
+description: 'In this post, we aim to articulate that vision and suggest a path to making it concrete, focusing on three libraries at the core of the PyData ecosystem: SciPy, scikit-learn and scikit-image. '
+category: [Array API]
+featuredImage:
+  src: ../public/posts/pydata-extensibility-vision/feature-image-scipy-ndimage-dispatcher.png
+  alt: 'A schema using SciPy dispatcher to allow users to develop generic code that works with arrays from different libraries.'
+hero:
+  imageSrc: ../public/posts/pydata-extensibility-vision/hero-image.png
+  imageAlt: 'Cropped diagram of proposed dispatch mechanism layers for enabling CuPy and Dask support.'
+---
+
+Over the years, array computing in Python has evolved to support distributed
+arrays, GPU arrays, and other various kinds of arrays that work with specialized
+hardware, or carry additional metadata, or use different internal memory
+representations. The foundational library for array computing in the PyData
+ecosystem is NumPy. But NumPy alone is a CPU-only library - and a
+single-threaded one at that - and in a world where it's possible to get a GPU or
+a CPU with a large core count in the cloud cheaply or even for free in a matter
+of seconds, that may not seem enough. For the past couple of years, a lot of
+thought and effort has been spent on devising mechanisms to tackle this problem,
+and evolve the ecosystem in a gradual way towards a state where PyData
+libraries can run on a GPU, as well as in distributed mode across multiple
+GPUs.
+
+We feel like a shared vision has emerged, in bits and pieces. In this post, we
+aim to articulate that vision and suggest a path to making it concrete,
+focusing on three libraries at the core of the PyData ecosystem: SciPy,
+scikit-learn and scikit-image. We are also happy to share that AMD has
+recognized the value of this vision, and is partnering with Quansight Labs to
+help make it a reality.
+
+With the set of dispatch layers sketched in this post, array-consuming
+libraries will be able to use multiple kinds of arrays, without having to have
+a hard dependency on all of those array libraries. We'd like to have a unified
+design, using the same tools applicable across SciPy, scikit-learn,
+scikit-image and other libraries. In this situation, it's best to have a design
+discussion spanning multiple projects.
+
+
+<p align="center">
+    <img
+     alt="A diagram showing possible interconnections that would be enabled by Array API."
+     src="../public/posts/pydata-extensibility-vision/nep-0047-library-dependencies.png">
+    <i><br>Possible interconnections enabled in this design, via the Array API standard and Uarray.</i>
```

Comment:
```suggestion
    <br><i>Possible interconnections enabled in this design, via the Array API standard and Uarray.</i>
```

Reply:
build error: `Unexpected closing tag `</i>`, expected corresponding closing tag for `<br>``